UI/BuildDesignatorWnd.cpp

```diff
@@ -966,6 +1010,8 @@ void BuildDesignatorWnd::CompleteConstruction() {
         boost::bind(static_cast<void (EncyclopediaDetailPanel::*)(const BuildingType*)>(&EncyclopediaDetailPanel::SetItem),  m_enc_detail_panel, _1));
     m_build_selector->DisplayShipDesignSignal.connect(
         boost::bind(static_cast<void (EncyclopediaDetailPanel::*)(const ShipDesign*)>(&EncyclopediaDetailPanel::SetItem), m_enc_detail_panel, _1));
+    m_build_selector->DisplayStockpileProjectSignal.connect(
+        boost::bind(static_cast<void (EncyclopediaDetailPanel::*)(const std::string&, bool)>(&EncyclopediaDetailPanel::SetText), m_enc_detail_panel, "PROJECT_BT_STOCKPILE_DESC", true));
```

Comment:
have a proper article, not a `SetText` call that probably will cause problems with other pedia functions.

Reply:
Not sure how to do this. Put up an issue for this.

Reply:
> Not sure how to do this. I already put up an issue for this.

I thought I might consider assigning the issue to myself and putting up a PR, but I couldn't find a corresponding issue.  I found 2029 and 2030, but they both look like they are for other aspects of the ProductionWnd UI.

Reply:
Oops, in my mind I already had done that. Here is the issue 2040: https://github.com/freeorion/freeorion/issues/2040